tests/target-browser/target_browser_test.go

```diff
@@ -927,48 +939,59 @@ var _ = Describe("Target Browser Tests", func() {
 				Expect(string(output)).Should(ContainSubstring(fmt.Sprintf("flag needs an argument: %s", flagOperationScope)))
 			})
 
-			It(fmt.Sprintf("Should succeed if flag %s is not provided", flagOperationScope), func() {
+			It(fmt.Sprintf("Should get both OperationScoped backup and clusterbackups if flag %s is not provided", flagOperationScope), func() {
 				args := []string{cmdGet, cmdBackup}
-				args = append(args, commonArgs...)
-				cmd := exec.Command(targetBrowserBinaryFilePath, args...)
-				_, err := cmd.CombinedOutput()
-				Expect(err).ShouldNot(HaveOccurred())
+				backupData := runCmdBackup(args)
+				Expect(len(backupData)).To(Equal(noOfBackupPlansToCreate * noOfBackupsToCreate * 2))
 			})
 
-			It(fmt.Sprintf("Should succeed if flag %s is given zero value", flagOperationScope), func() {
-				args := []string{cmdGet, cmdBackup, flagOperationScope, ""}
-				args = append(args, commonArgs...)
-				cmd := exec.Command(targetBrowserBinaryFilePath, args...)
-				_, err := cmd.CombinedOutput()
-				Expect(err).ShouldNot(HaveOccurred())
+			It(fmt.Sprintf("Should get both backups and clusterbackups if flag %s is given zero value", flagOperationScope), func() {
+				args := []string{cmdGet, cmdBackup}
+				backupData := runCmdBackup(args)
+				Expect(len(backupData)).To(Equal(noOfBackupPlansToCreate * noOfBackupsToCreate * 2))
 			})
 
-			It(fmt.Sprintf("Should get %d backupPlan if flag %s is given %s value", noOfBackupPlansToCreate,
+			It(fmt.Sprintf("Should get %d backup if flag %s is given %s value", noOfBackupPlansToCreate,
 				flagOperationScope, internal.MultiNamespace), func() {
 				args := []string{cmdGet, cmdBackup, flagOperationScope, internal.MultiNamespace}
 				backupData := runCmdBackup(args)
 				Expect(len(backupData)).To(Equal(noOfBackupPlansToCreate))
```

Comment:
noOfBackupPlansToCreate * noOfBackupsToCreate

update this, wherever applicable